scripts/link-ts-references.ts

```diff
@@ -129,7 +130,7 @@ function saveConfigs(configs: PackageInfo[]) {
   configs.forEach(({ modified, path, tsConfig }) => {
     if (modified) {
       const tsConfigFile = join(path, "tsconfig.json");
-      writeFileSync(tsConfigFile, JSON.stringify(tsConfig, null, 2));
+      writeFileSync(tsConfigFile, JSON5.stringify(tsConfig, null, 2));
```

Comment:
This `link-ts-references.ts` script was previously clobbering comments in `tsconfig.json` files. This JSON5 library fixes it :mage_man:. (RIP old comments)